Task6/AbstractFactory/PaperBookAbstractFactory.cs

```diff
@@ -0,0 +1,145 @@
+﻿using Microsoft.Extensions.FileSystemGlobbing.Internal;
+using Microsoft.VisualBasic.FileIO;
+using System.Text.RegularExpressions;
+
+namespace Task6.AbstractFactory
+{
+    internal class PaperBookAbstractFactory : ILibrayAbstractFactory
+    {
+        List<string> _uniquePublishers;
```

Comment:
Can't understand the purpose of this value, at least a comment is needed

Reply:
Modified